server2/requirements.txt

```diff
@@ -0,0 +1,5 @@
+Flask==1.0.2
```

Comment:
If you're going to hard code requirements.txt (which makes sense so things don't break), you should probably have a travis.yml line for doing a security scans. it's too easy to forget about these and have a problem down the road.

Reply:
Done.